performance-test/README.md

```diff
@@ -0,0 +1,98 @@
+# performance-test
```

Comment:
Maybe it's more helpful to document how this project should be used, otherwise I suggest removing it as these template doesn't add much information for us.